src/lib/support/Span.h

```diff
@@ -42,9 +42,14 @@ class Span
     using reference = T &;
 
     constexpr Span() : mDataBuf(nullptr), mDataLen(0) {}
-    constexpr Span(pointer databuf, size_t datalen) : mDataBuf(databuf), mDataLen(datalen) {}
+
+    Span(pointer databuf, size_t datalen) : mDataBuf(databuf), mDataLen(datalen)
```

Comment:
There's a comment further down that `VerifyOrDie` is incompatible with `contstexpr` on some platforms. I think it's generally fine for this constructor to not be constexpr since constexpr use cases in practice seem to be based on arrays and therefore can easily use the constructor without an explicit length.

I'll see if adding that overload works.

Reply:
Added `nullptr_t` overloads to both Span and FixedSpan.